test/test_scala_library_outputs_mixed_java_scala_jars.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echoerr() {
+  echo "$@" 1>&2;
+}
+
+assert() {
+  $@ || (echo "FAILED: $@"; exit 1)
+}
+
+contains() {
+  grep $@
+}
+
+set -e
+
+TARGET="MixJavaScalaLib"
```

Comment:
Really small comment: can this also move to parameter? Symmetry wise it feels like it should really sit in the sh_test

Reply:
Done!